library/core/src/option.rs

```diff
@@ -807,12 +839,12 @@ impl<T> Option<T> {
     /// ```
     #[inline]
     #[stable(feature = "option_entry", since = "1.20.0")]
-    pub fn get_or_insert(&mut self, v: T) -> &mut T {
-        self.get_or_insert_with(|| v)
+    pub fn get_or_insert(&mut self, value: T) -> &mut T {
```

Comment:
I don't think we should change this in this patch, it could be done in another pull request.

Reply:
These changes are in a separate commit. I think it's fine to merge this commit as part of this PR to save some hassle.